autogpt_platform/VIRUS_SCANNING.md

```diff
@@ -0,0 +1,267 @@
+# ClamAV Virus Scanning Integration
+
```

Comment:
Include this as a relative import in docs via mkdocs.yml

Reply:
Include as what information?

Reply:
I think it’s not a bad pattern just throw it in the docs/autogpt_platform as a contributing guide